index.rst

```diff
@@ -127,8 +127,8 @@ Dates in *italic* are scheduled and can be adjusted.
 
 By default, the end-of-life is scheduled 5 years after the first release,
 but can be adjusted by the release manager of each branch.  The support for
-Python 2.7 has currently been extended to 2020-01-01.  Versions older than
-2.7 have reached end-of-life.
+Python 2.7 was extended to 2020-01-01.  All Python 2 versions have reached
+end-of-life.
```

Comment:
Once 2.7 in moved to complete EOL, the 2.7-specific extension sentence should be removed.  But I think the Py 2 EOL sentence should be left.